guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -25,40 +25,47 @@ ifdef::foreman-el,katello[]
 include::proc_configuring-repositories-el.adoc[]
 
 +
-. Enable Ruby 2.7 module:
+. If the Ruby 2.5 module has already been enabled, you need to perform a module reset.
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # {package-manager} module reset ruby
-# {package-manager} module enable ruby:2.7
 ----
-endif::[]
 
-ifdef::katello[]
+. If the PostgreSQL 10 module has already been enabled, you need to perform a module reset.
```

Comment:
Same as above, but additionally: if the user actually has PostgreSQL 10 installed and *data* in there, 12 will not be able to start with that data.

I am not sure if this file is also included in *upgrade* docs, but if it is, this would require additional clarification.

Reply:
See what I wrote in the commit message/PR description. Right now the upgrade docs completely rely on foreman-maintain. I have no clue how that's going to handle things. Or if that even works in upstream.

Reply:
Right, I missed that.

I think f-m doesn't support upstream upgrades yet.

Reply:
Which also makes me wonder: reading https://docs.theforeman.org/nightly/Upgrading_and_Updating/index-katello.html it looks like it does install the new repositories but it doesn't talk about EL8 at all.